_layouts/event.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+---
+
+<main>
```

Comment:
I am not sure the `<main>` is needed here

Reply:
Fair point. 😅 Removed. Thanks!